resources/views/settings/index.blade.php

```diff
@@ -444,11 +444,13 @@
                 }
             });
 
-            $('.voyager-trash').click(function () {
+            $('.panel-actions .voyager-trash').click(function () {
                 var display = $(this).data('display-name') + '/' + $(this).data('display-key');
-
+				var form = $('#delete_form')[0];
+				
                 $('#delete_setting_title').text(display);
-                $('#delete_form')[0].action = $('#delete_form')[0].action.replace('__id', $(this).data('id'));
+                form.action = form.action.match(/\/[0-9]+$/) ? form.action.replace(/([0-9]+$)/, $(this).data('id')) : form.action + '/' + $(this).data('id');
```

Comment:
This fix doesn't support non-numeric id values.  While we don't explicitly make use of them within Voyager, there's nothing saying we won't in the future